src/components/WwwFrame/PromotionalBanner/Banners/AppealBanner/AppealBanner15.vue

```diff
@@ -145,169 +82,70 @@ export default {
 	data() {
 		return {
 			open: true,
-			amount: 0,
-			donationAmount: null,
-			percentTowardGoal: null,
-			// GROW-230
-			forceDismissOverlayExperiment: { id: 'homepage_force_dismiss_overlay', version: 'variant-a' },
+			fifteenYearGoalPercent: 0
 		};
 	},
 	apollo: {
-		query: appealBannerQuery,
+		query: recentFundedLoans,
 		preFetch: true,
+		preFetchVariables() {
+			return {
+				start: '2020-10-01'
+			};
+		},
 		result({ data }) {
-			this.amountRaised = _get(data, 'general.kivaStats.latestDonationCampaign.amount_raised');
-			this.targetAmount = _get(data, 'general.kivaStats.latestDonationCampaign.target_amount');
+			const numRecentFundedLoans = data?.general?.kivaStats?.numRecentFundedLoans || 0;
+			if (numRecentFundedLoans !== 0) {
+				this.fifteenYearGoalPercent = Math.round(
+					(numRecentFundedLoans / 15000) * 100
```

Comment:
The 15000 value needs to come out of the Contentful banner data object. It is subject to change throughout the month especially if we reach it quickly.

Reply:
👍